section/doc/tactic-developer/developer/packaging-python-scripts-into-a-plugin/index.txt

```diff
@@ -3,30 +3,32 @@ TACTIC - Packaging Python Scripts into a Plugin
 
 *Creating Plugin for Scripts*
 
-1) Prepare a folder or folder structure containing all of the python scripts you would like to include in the plugin. 
+1.  Prepare a folder or folder structure containing all of the python scripts you would like to include in the plugin. 
 
-The scripts can be contained within any folder level in the hierarchy. The plugin will allow you to have access to every one of those scripts.
 
-2) Copy your folder of scripts.
```

Comment:
When generating the books, this comes out as item #1 and the rest follow from here i.e. #2, #3, etc. This should be item #2. It looks like it's set correctly here, but please make sure it comes out correctly in the generated books.